source/jormungandr/tests/departure_board_tests.py

```diff
@@ -26,38 +26,37 @@
 # IRC #navitia on freenode
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
+from collections import namedtuple
+import itertools
 from tests_mechanism import AbstractTestFixture, dataset
 from check_utils import *
 import datetime
 
 
-def check_departure_board(schedules, tester, only_time=False):
+def is_valid_stop_schedule(schedules, tester, only_time=False):
     """
     check the structure of a departure board response
     """
-    assert len(schedules) == 1, "there should be only one elt"
+    for schedule in schedules:
+        is_valid_stop_point(get_not_null(schedule, "stop_point"), depth_check=2)
 
-    schedule = schedules[0]
+        route = get_not_null(schedule, "route")
+        is_valid_route(route, depth_check=2)
 
-    is_valid_stop_point(get_not_null(schedule, "stop_point"), depth_check=2)
+        datetimes = get_not_null(schedule, "date_times")
 
-    route = get_not_null(schedule, "route")
-    is_valid_route(route, depth_check=2)
-
-    datetimes = get_not_null(schedule, "date_times")
+        assert len(datetimes) != 0, "we have to have date_times"
+        for dt_wrapper in datetimes:
+            dt = dt_wrapper["date_time"]
+            if only_time:
+                get_valid_time(dt)
+            else:
+                get_valid_datetime(dt)
+            #TODO remove href_mandatory=False after link refactor, they should always be there :)
```

Comment:
this is an old comment (the function has just been moved)

And the PR cannot be prepared just like that. A link refactoring would be quite a big refactoring, and it will be difficult not break the API during this (so my guess is that it will wait for the v2 :smile: )